internal/pkg/api/storageapi/error.go

```diff
@@ -25,7 +21,7 @@ type Error struct {
 
 func (e *Error) Error() string {
 	req := e.response.Request
-	msg := fmt.Sprintf(`%s, method: "%s", url: "%s", httpCode: "%d"`, e.Message, req.Method, req.URL, e.HttpStatus())
+	msg := fmt.Sprintf(`%s, method: "%s", url: "%s", httpCode: "%d"`, e.Message, req.Method, req.URL, e.StatusCode())
```

Comment:
Goa vyzaduje nazov metody, `StatusCode`, a nechcel som mat 2 metody, co robia to iste.